cloud: support backup using EBS snapshots (#12862)
pingcap/docs: tidb-cloud/backup-and-restore.md

    - In **Backup Retention**, configure the minimum backup data retention period.

    - Turn on or off the **EBS snapshot** feature (beta).

        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
        
        - Minimize the impact of backup on cluster performance, with the potential to keep the impact on QPS and transaction latency under 1% while using no cluster CPU or memory resources.
        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.

      To use the EBS snapshot feature, make sure that the following prerequisites are met:

            - Your TiDB cluster is hosted on AWS and your TiDB version is v6.5.1 or later.

>             - When you restore the data backed up by the **EBS snapshot** feature, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.

Author: ```suggestion
            - When you restore the data backed up by the **EBS snapshot** feature to a new cluster, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
```

>             - Your TiDB cluster is hosted on AWS and your TiDB version is v6.5.1 or later.
>             - When you restore the data backed up by the **EBS snapshot** feature to a new cluster, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
>             - The EBS snapshot feature and PITR feature are mutually exclusive. Therefore, only one of them can be enabled at a time.

Author: ```suggestion
      - Your TiDB cluster is hosted on AWS and your TiDB version is v6.5.1 or later.
       - When you restore the data backed up by the **EBS snapshot** feature to a new cluster, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
       - The EBS snapshot feature and PITR feature are mutually exclusive. Therefore, only one of them can be enabled at a time.
```

    - In **Backup Retention**, configure the minimum backup data retention period.

>     - Turn on or off the **EBS snapshot** feature.

Author: ```suggestion
    - Turn on or off the **EBS snapshot** feature (beta).
```

> 

Author: Question: Do we need to mention this feature is in beta and not recommended for production environments?

Reviewer: I do not think so.

        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
        

>         - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.

Reviewer 2: "nearly eliminate" a little strange to me, what about"Minimize the impact on cluster performance during backup, with the potential to keep the impact on QPS and transaction latency under 1%, while not utilizing any cluster CPU or memory."?

>         - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.

Author: ```suggestion
        - Minimize the impact of backup on cluster performance, with the potential to keep the impact on QPS and transaction latency under 1% while using no cluster CPU or memory resources.
```

>         The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
>         
>         - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency less than 1%, and not consuming any cluster CPU or memory.
>         - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.

Reviewer 3: ```suggestion
        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
        
        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.
        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
```

>       To use the EBS snapshot feature, make sure that the following requirements are met:

Reviewer 2: ```suggestion
      To use the EBS snapshot feature, make sure that the following prerequisites must be met:
```

Author: prerequisites are met 就可以了。 prerequisites 本身就是 something that is necessary before something else can happen or be done

>             - Your TiDB cluster is hosted on AWS and your TiDB version is at least v6.5.0. 

Reviewer 2: ```suggestion
            - The TiDB cluster is hosted on AWS and has a minimum TiDB version of v6.5.0. 
```

Author: Your 是和前面的 You can use it to 对应。 At least 为了和 PITR 里的描述尽量一致，at least v6.5.0 我可以改成 v6.5.0 or later.

>             - Your TiDB cluster is hosted on AWS and your TiDB version is at least v6.5.0. 

Author: ```suggestion
            - Your TiDB cluster is hosted on AWS and your TiDB version is v6.5.1 or later
```

>             - If your TiDB version is v6.5.0, the TiKV node size is at least 4 vCPU and 16 GiB. 

Author: ```suggestion
            - If your TiDB version is v6.5.0, the TiKV node size is at least 4 vCPU and 16 GiB. If your TiDB version is v6.5.1 or later, there is no requirement on the TiKv node size.
```

>             - If your TiDB version is v6.5.0, the TiKV node size is at least 4 vCPU and 16 GiB. 

Reviewer 2: ```suggestion
            - If the TiDB version is v6.5.0, the TiKV node config must be a minimum of 4 vCPU and 16 GiB.
```

>             - If your TiDB version is v6.5.0, the TiKV node size is at least 4 vCPU and 16 GiB. If your TiDB version is v6.5.1 or later, there is no requirement on the TiKV node size.

Author: ```suggestion
```

>             - When you restore the data backed up by the **EBS snapshot** feature, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.

Reviewer 2: ```suggestion
            - When restoring the data was backed up by the **EBS snapshot** feature, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
```

Author: 这里只能是 When you restore，因为当使用 When doing xxx, sb/sth xx 时候的时候，两者的主语必须相同，否则就是垂悬结构语法错误。可参见这里 https://baike.baidu.com/item/%E6%82%AC%E5%9E%82%E7%BB%93%E6%9E%84/3924722?fr=aladdin

        - Resolve cases of data write errors by restoring point-in-time that is before the error event.
        - Audit the historical data of the business.

>         To use the PITR feature, make sure that your TiDB cluster version is at least v6.4.0, the TiKV node size is at least 8 vCPU and 16 GiB, and the EBS snapshot feature is disabled.

Author: ```suggestion
        To use the PITR feature, make sure that your TiDB cluster version is v6.4.0 or later, the TiKV node size is at least 8 vCPU and 16 GiB, and the EBS snapshot feature is disabled.
```
